.gitignore

```diff
@@ -1,9 +1,4 @@
-*.swp
-*.beam
-.DS_Store
 .bundle
-.config
-.ruby-version
```

Comment:
Removed this, as it deals with version managers, and we should not be concerned with what exact version they are running, just a minimum version as communicated in the Track readme.  We don't "enforce" it.